core/domain/skill_services.py

```diff
@@ -777,10 +777,8 @@ def delete_skill(committer_id, skill_id, force_deletion=False):
             still retained in the datastore. This last option is the preferred
             one.
     """
-    skill_model = skill_models.SkillModel.get(skill_id)
-    skill_model.delete(
-        committer_id, feconf.COMMIT_MESSAGE_SKILL_DELETED,
-        force_deletion=force_deletion)
+    skill_models.SkillModel.delete_multi(
+        [skill_id], committer_id, '', force_deletion=force_deletion)
```

Comment:
Could you please add tests for deleting partially deleted models?

Reply:
Ditto for questions

Reply:
Done.